e2e/fixtures/createClaim.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
       }
     },
     claimant: {
+      allocatedTrack: 'SMALL_CLAIM',
```

Comment:
We expect allocatedTrack to be set by service callback, so we shouldn't set it via data store request

Reply:
As the same is being used by api tests, and we need this for apit test as well.

Reply:
done